build.gradle

```diff
@@ -22,7 +22,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.0-rc02'
+        classpath 'com.android.tools.build:gradle:3.3.0-alpha08'
```

Comment:
I had problems earlier today by using alpha version of gradle on circle CI. I would keep the previous version.